### DISCUSSION TITLE
Extruder skipping

### DISCUSSION THREAD

**Discussion by palehorse102** [Jyers/Marlin#204] [Q&A] [answered]
I flashed the precompiled Default V4 file and while trying calibrate the e steps the speed of extrusion was so fast that it was causing the gear to skip. Reflashed to stock and everything is good to go. Any ideas?

**Comment by Jyers** [comment]
The extrusion feedrate for the move menu is 5 mm/s, which should be easily handled by most setups, perhaps try increasing your temps. If it seems to be going significantly faster than 5 mm/s than we can look into it to see if maybe its a bug of some kind.

**Reply by tnagy294** [reply]
I noticed a similar problem too, flashed the original creality FW 1.0.2 and no gear skipping. I compiled myself but haven't changed anything I don't know exactly what it does....

**Reply by palehorse102** [reply]
> The extrusion feedrate for the move menu is 5 mm/s, which should be easily handled by most setups, perhaps try increasing your temps. If it seems to be going significantly faster than 5 mm/s than we can look into it to see if maybe its a bug of some kind.


Thanks for getting back to me and I am sorry for not replying sooner I was in the middle of a long run of prints I needed to finish. 

I ran a few timed extrusion tests using the stock firmware and the precompiled Default V4.  My e-steps were set to 102 for both tests and no filament was loaded.  All other settings were set to the default values. 

The stock firmware extruded 100mm in 115 seconds, while V4 extruded in 23 seconds. I ran one test on the stock firmware with filament and it took 120 seconds for 100mm, I did not repeat this test on the V4 as I did not want to clog my hotend again.

**Reply by Jyers** [reply]
Thanks for the update, I'll do some testing and probably lower the extrusion speed a bit, it might still be slightly faster than the stock as I think stock might be slower the necessary, but I'll make sure it is slow enough to not skip

**Comment by Jyers** [comment, marked as answer]
Update, the feedrate has been reduced to 2 mm/s, this will be part of the next release

**Reply by Schnabulation** [reply]
I actually never had any problems with extruding from the menu. What is the stock Marlin (not stock Creality-FW) feedrate for the E move?

**Reply by Jyers** [reply]
Actually looking into the code I found that marlin does have a set of preset feedrates for manual moves, I will use those instead. This will make the E move feedrate 4 mm/s, but hopefully that's still slow enough not to skip in most cases
